tools/gulp/tasks/docs.ts

```diff
@@ -86,19 +100,19 @@ task('markdown-docs', () => {
     }
   };
 
-  return src(['src/lib/**/*.md', 'src/cdk/**/*.md', 'guides/*.md'])
-      .pipe(markdown({
-        // Add syntax highlight using highlight.js
-        highlight: (code: string, language: string): string => {
-          if (language) {
-            // highlight.js expects "typescript" written out, while Github supports "ts".
-            let lang = language.toLowerCase() === 'ts' ? 'typescript' : language;
-            return hljs.highlight(lang, code).value;
-          }
-
-          return code;
-        }
-      }))
+  return src(['src/lib/**/*.md', 'guides/*.md'])
+      .pipe(rename({prefix: 'material-'}))
+      .pipe(markdown(markdownOptions))
+      .pipe(transform(transformMarkdownFiles))
+      .pipe(dom(createTagNameAliaser('docs-markdown')))
+      .pipe(dest('dist/docs/markdown'));
+});
+
+/** Generates html files from the markdown overviews and guides. */
+task('markdown-docs-cdk', () => {
```

Comment:
@crisbeto or @DevVersion I know this is wrong but I don't know the right way to fix it

(disambiguating material and cdk output)

Reply:
Yeah I see. Would have been nice to name the other task `markdown-docs-material` or something, but this is not really a priority right now.

Reply:
Yeah I see. Would have been nice to name the other task `markdown-docs-material` or something, but this is not really a priority right now.

Reply:
Not sure a better way to resolve this right now, left a TODO